buildlib/pr/wire_compat.yml

```diff
@@ -90,22 +90,28 @@ jobs:
           UCX_LEGACY_PATH: $(System.DefaultWorkingDirectory)/$(ucx_legacy)
         displayName: compare tls performance characteristics
         condition: ne(variables.ucx_tag, 'v1.14.0')
-      - bash: buildlib/tools/test_wire_compat.sh $(test_dir)/client_server -a "-c tag"
+      - bash: |
+          export WIRE_COMPAT_TEST_PORT=$((10000 + 1000 * ${AZP_AGENT_ID}))
+          buildlib/tools/test_wire_compat.sh $(test_dir)/client_server -a "-c tag"
         env:
           UCX_PR_LIB_PATH: $(System.DefaultWorkingDirectory)/$(ucx_current)/lib
           UCX_LEGACY_LIB_PATH: $(System.DefaultWorkingDirectory)/$(ucx_legacy)/lib
           AZP_AGENT_ID: $(AZP_AGENT_ID)
         displayName: ucp_client_server with UCX $(ucx_tag)
         condition: ne(variables.ucx_tag, 'v1.14.0')
         timeoutInMinutes: 2
-      - bash: buildlib/tools/test_wire_compat.sh $(test_dir)/ucp_hworld -n
+      - bash: |
+          export WIRE_COMPAT_TEST_PORT=$((10000 + 2000 * ${AZP_AGENT_ID}))
```

Comment:
Is it an issue that it will be the same port number for AZP_AGENT_ID equal to 0? The same question is for `10000 + 1000 * ${AZP_AGENT_ID}`, and `10000 + 2000 * ${AZP_AGENT_ID}` for agents 2 and 1 respectively.

Reply:
AZP_AGENT_ID values are starting from 1 (checked in our azure settings)

Reply:
Ok, what about `10000 + 1000 * ${AZP_AGENT_ID}`, and `10000 + 2000 * ${AZP_AGENT_ID}` for agents 2 and 1 respectively;  `10000 + 1000 * ${AZP_AGENT_ID}`, and `10000 + 3000 * ${AZP_AGENT_ID}` for agents 3 and 1; etc?

Reply:
when this yaml is being processed, AZP_AGENT_ID remains the same. I have to make sure that different scripts executed in one job use different ports. Since AGEND_ID is the same I pass different multiplicators (1000, 2000, etc)

Reply:
I see.
I was worried about the case when two different instances with different ids are processed on the same machine.

Reply:
in such situation they have different (AZP_AGENT_IDs starting from 1)